src/client/keycode.cpp

```diff
@@ -6,15 +6,17 @@
 #include "settings.h"
 #include "log.h"
 #include "debug.h"
+#include "renderingengine.h"
 #include "util/hex.h"
 #include "util/string.h"
 #include "util/basic_macros.h"
+#include <vector>
 
 struct table_key {
-	const char *Name;
+	std::string Name;
 	irr::EKEY_CODE Key;
 	wchar_t Char; // L'\0' means no character assigned
-	const char *LangName; // NULL means it doesn't have a human description
+	std::string LangName; // NULL means it doesn't have a human description
```

Comment:
comment wrong

Reply:
Corrected in 9590be17f121b09c78bd9f1d28dba0603049ec29.